File: src/main/java/org/intellij/sdk/language/jass/psi/JassTokenType.java
package org.intellij.sdk.language.jass.psi;

import com.intellij.psi.tree.IElementType;
import guru.xgm.jass.lang.Language_JASS;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class JassTokenType extends IElementType {

    public JassTokenType(@NotNull @NonNls String debugName) {
        super(debugName, Language_JASS.INSTANCE);
    }

    @Override
    public String toString() {
        return "JassTokenType." + super.toString();
    }
}

File: src/main/kotlin/raft/war/language/jass/psi/JassTokenType.kt
package raft.war.language.jass.psi

import com.intellij.psi.tree.IElementType
import raft.war.language.jass.lang.JassLanguage.Companion.instance
import org.jetbrains.annotations.NonNls

class JassTokenType(debugName: @NonNls String) : IElementType(debugName, instance){
    override fun toString() = "JassTokenType.${super.toString()}"
}
